### ISSUE TITLE
Error creating orcpub docker image

### ISSUE THREAD

**Issue by Craxter** [Orcpub/orcpub#539]
**Describe the bug**
When creating the docker image using the docker-compose-build.yaml an error appears when trying to retrieve pdfbox from repository

**To Reproduce**

1. Download develop branch
2. Modify docker-compose-build.yaml to add DATOMIC_URL, SIGNATURE, ADMIN_PASSWORD and DATOMIC_PASSWORD
3. from terminal launch docker-compose -f docker-compose-build.yaml up
4. Error will appears in step "RUN lein deps" while trying to create the orcpub image:

```
Could not find artifact org.apache.pdfbox:pdfbox:jar:2.1.0-20170324.170253-831 in clojars (https://repo.clojars.org/)
Could not find artifact org.apache.pdfbox:pdfbox:jar:2.1.0-20170324.170253-831 in  (https://repository.apache.org/snapshots/)
Could not transfer artifact org.apache.pdfbox:pdfbox:jar:2.1.0-20170324.170253-831 from/to my.datomic.com (https://my.datomic.com/repo): Not authorized
Failed to read artifact descriptor for org.apache.pdfbox:pdfbox:jar:2.1.0-20170324.170253-831
This could be due to a typo in :dependencies, file system permissions, or network issues.
If you are behind a proxy, try setting the 'http_proxy' environment variable.
```

**Desktop (please complete the following information):**

 - OS: Windows 10 21H1 version
 - Version: 2.5.0.22

**Comment by codeGlaze** [comment]
We just discovered that version of pdfbox has been pulled from the official repo for some reason.

Switching to another version *should* work.

**Comment by kapsterman** [comment]
I changed a line in project.clj :dependencies from '[org.apache.pdfbox/pdfbox "2.1.0-20170324-831"]' to: '[org.apache.pdfbox/pdfbox "2.0.24"]' and the containers build without error.  However, the "Print to PDF" functionality of the webserver is now broken.  I rebuilt the containers with every available version of PDFBox (from 2.0.0 to 2.0.24) and none of the other versions could produce a PDF from the resulting website. The containers will not build with any version of PDFBox before 2.0.0.
